### PR TITLE
Team doc

### PR DESCRIPTION
**Pull request by lemonmade**
This PR outlines the Web Foundation team's responsibilities and ways of working. It mostly came out of me wishing I had a whole summit talk just for us, so it fills in many of the gaps that we talked about on stage.

Sorry, I know it's long, but I want this to serve as a contract we can point people to.

### CODE REVIEW

File: Meta/Web Foundations team responsibilities.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+The Web Foundation team applies this same principle of having sensible "picks" to developer tools, application libraries, and best practices specific to the Web platform. However, we are mindful of not creating "thin clients" — that is, leaving no decisions to application teams about what their application should look like.
+
+It is useful to consider the separation of powers between federal and provincial governments, as this is a similar model to how the Web Foundation team considers topics in which to involve themselves. In Canada, the federal government delegates the responsibility for overseeing schools, hospitals, and property rights to provincial governments, while retaining powers that they are best positioned to address, such as national defence and currency. Similarly, the Web Foundations team is responsible for guiding decisions on topics that are important for all applications, but delegates most of the finer points to application teams.
```
Comment (tzvipm) [inline Meta/Web Foundations team responsibilities.md]: Not sure how useful this paragraph is.

File: Meta/Web Foundations team responsibilities.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+### Criteria for Web Foundation involvement
+
+The above description can be seen as overly broad, so the Web Foundation team uses the following criteria in deciding whether it should take a role in deciding best practices or preferred tools for a particular topic.
+
+The following criteria must **always** be met for the Web Foundation team to get involved:
+
+* It **must not** interfere with a team’s ability to deliver value to their user. This has two implications:
+  1. The Web Foundation team only builds and maintains libraries that are either non-visual for the end user, or which operate in line with our design system, Polaris.
+  2. Any team feeling that our recommended best practices or libraries prevent them from being capable of delivering a user-facing feature should ignore the recommendation and bring this up with us.
```
Comment (tzvipm) [inline Meta/Web Foundations team responsibilities.md]: maybe bold the work `and` here?

File: Meta/Web Foundations team responsibilities.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* It **must** be a topic that affects a majority of applications. The Web Foundation team will not build libraries for that benefit only a small handful of projects.
```
Comment (tzvipm) [inline Meta/Web Foundations team responsibilities.md]: `for that` -> `that`

Comment (tzvipm) [reply Meta/Web Foundations team responsibilities.md]: Also, what do we mean by `small handful`? Don't we build such tools for `core` on occasion? Perhaps rephrase this to mention scope rather than number of projects?

File: Meta/Web Foundations team responsibilities.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+These responsibilities are summarized in the diagram below.
+
+<figure>
+  <img src="./images/Responsibilities%20-%20Shopify.png" alt="Web Foundation responsibilities for Shopify" />
+  <figcaption>A summary of components that the Web Foundation team is responsible for (the bottom four layers, highlighted in blue)</figcaption>
+</figure>
+
+## Maintenance of foundational piece of Shopify Web
+
+![Web Foundation responsibilities for Shopify Web](./images/Responsibilities%20-%20Web.png)
```
Comment (tzvipm) [inline Meta/Web Foundations team responsibilities.md]: does this need a `figcaption` as well for the asterisk next to `Polaris`?

File: Meta/Web Foundations team responsibilities.md (hunk 1 above)
Comment (cartogram) [reply Meta/Web Foundations team responsibilities.md]: I agree here. I personally like the analogy but fear others may not.

File: Meta/Web Foundations team responsibilities.md (hunk 4 above)
Comment (cartogram) [reply Meta/Web Foundations team responsibilities.md]: Was wondering about the asterisk as well.

File: Meta/Web Foundations team responsibilities.md (hunk 1 above)
Comment (lemonmade) [reply Meta/Web Foundations team responsibilities.md]: :sad_panda:
